boards/nucleo-l073/include/periph_conf.h

```diff
@@ -214,14 +214,14 @@ static const spi_conf_t spi_config[] = {
 #define I2C_1_EVT_IRQ       I2C3_IRQn
 #define I2C_1_EVT_ISR       isr_i2c3
 /* I2C 1 pin configuration */
-#define I2C_1_SCL_PORT      PORT_A
-#define I2C_1_SCL_PIN       8
-#define I2C_1_SCL_AF        3
-#define I2C_1_SCL_CLKEN()   (periph_clk_en(AHB, RCC_IOPENR_GPIOAEN))
-#define I2C_1_SDA_PORT      PORT_B
-#define I2C_1_SDA_PIN       5
-#define I2C_1_SDA_AF        8
-#define I2C_1_SDA_CLKEN()   (periph_clk_en(AHB, RCC_IOPENR_GPIOBEN))
+#define I2C_1_SCL_PORT      PORT_C
```

Comment:
Thanks for updating this. Unfortunately, I'm still not convinced by the 3rd channel using PC0/1 because it will conflict with ADC pins. With #7277, this may happen soon.
I'm fine with keeping the possibility to configure 3 channels but only keep 2 configured by default in the periph_conf.